### PR TITLE
Added LCM user properties (v1.0 endpoint)

### PR DESCRIPTION
**Pull request by cmmdesai**
Added new attributes to user object - employeeType, employeeOrgData and employeeHireDate

### CODE REVIEW

File: api-reference/v1.0/resources/employeeOrgData.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "employeeOrgData resource type"
```
Comment (Lauragra) [inline api-reference/v1.0/resources/employeeOrgData.md:2]: File names need to be all lowercase - please update.

Comment (Lauragra) [inline api-reference/v1.0/resources/employeeOrgData.md:2]: This complex type does not exist in beta. We don't update v1.0 without updating beta first. Does this need to be documented for beta?

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please see comments. This PR will also require an update to the changelog.

File: api-reference/v1.0/resources/user.md [hunk 2]
```diff
@@ -125,13 +125,16 @@ This resource supports:
...
 |deletedDateTime| DateTimeOffset | The date and time the user was deleted. <br><br>Returned only on $select. |
 |department|String|The name for the department in which the user works. Supports $filter.|
 |displayName|String|The name displayed in the address book for the user. This is usually the combination of the user's first name, middle initial and last name. This property is required when a user is created and it cannot be cleared during updates. Supports $filter and $orderby.|
+|employeeHireDate| DateTimeOffset | The date and time when the user was hired or will start work in case of a future hire. <br><br>Returned only on $select. Supports $filter.|
 |employeeId|String|The employee identifier assigned to the user by the organization. Supports $filter.|
+|employeeOrgData|[employeeOrgData](employeeOrgData.md) collection|Represents organization data (e.g. division and costCenter) associated with a user.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/user.md:136]: Do you need to update response examples in operation topics to include the new property?

File: api-reference/v1.0/resources/employeeOrgData.md (hunk 1 above)
Comment (cmmdesai) [reply api-reference/v1.0/resources/employeeOrgData.md]: Done.

Comment (cmmdesai) [reply api-reference/v1.0/resources/employeeOrgData.md]: Done. Here is the beta update PR - https://github.com/microsoftgraph/microsoft-graph-docs/pull/9790

File: api-reference/v1.0/resources/user.md (hunk 2 above)
Comment (cmmdesai) [reply api-reference/v1.0/resources/user.md]: Since these are optional properties, we don't need to update the response examples for different operations.

**Review by alok-bansal — APPROVED** [review]

**Comment by alok-bansal** [comment]
> Please see comments. This PR will also require an update to the changelog.

Updated.

**Review by davidmu1 — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**